sycl/test/regression/ambigious-memcpy.cpp

```diff
@@ -0,0 +1,16 @@
+// detail::memcpy, even though in a different namespace, can cause ambiguity with libc's memcpy, due to argument dependent lookup (ADL). This compile-only test checks for ambigiouity in call to detail::memcpy.
+
+// RUN: %clang -fsycl -fsycl-device-only %s -o %s.out
```

Comment:
I think that you can limit this to `-fsyntax-only` and therefore drop `-o`

Reply:
Fixed in 87f8f3ea47d9c9ec5ea65471fca9bb168f2b262f